playlists/utils.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+from playlists.models import Playlist
+
+
+class PlaylistUtils:
+    def create_random(self, user):
+        result_playlist = Playlist(
+            name="True Random", description="Created with Derek's Spotify Tool"
```

Comment:
Someday, I'll show you how to do flake8 plugins. One of the ones I'm quite fond of are requiring trailing commas. Between that flake8 plugin and `black`, this would end up putting each argument on it's own line. Not required. Just a readability thing.